policy/modules/services/tor.te

```diff
@@ -119,6 +135,13 @@ tunable_policy(`tor_bind_all_unreserved_ports',`
 	corenet_tcp_bind_all_unreserved_ports(tor_t)
 ')
 
+optional_policy(`
+	tunable_policy(`tor_pluggable_transport_obfs4proxy',`
+		obfs4proxy_domtrans(tor_t)
+		obfs4proxy_signal(tor_t)
+	')
+')
```

Comment:
Does using obfs4proxy present a security problem for tor? It seems to me that this should simply be optional otherwise.